rest_framework-stubs/serializers.pyi

```diff
@@ -62,8 +62,17 @@ from rest_framework.relations import RelatedField as RelatedField
 from rest_framework.relations import SlugRelatedField as SlugRelatedField
 from rest_framework.relations import StringRelatedField as StringRelatedField
 from rest_framework.utils.model_meta import FieldInfo, RelationInfo
-from rest_framework.utils.serializer_helpers import BindingDict, BoundField, ReturnDict, ReturnList
-from rest_framework.validators import BaseUniqueForValidator, UniqueTogetherValidator, Validator
+from rest_framework.utils.serializer_helpers import BindingDict as BindingDict
+from rest_framework.utils.serializer_helpers import BoundField as BoundField
+from rest_framework.utils.serializer_helpers import JSONBoundField as JSONBoundField
+from rest_framework.utils.serializer_helpers import NestedBoundField as NestedBoundField
+from rest_framework.utils.serializer_helpers import ReturnDict as ReturnDict
+from rest_framework.utils.serializer_helpers import ReturnList as ReturnList
+from rest_framework.validators import BaseUniqueForValidator, Validator
+from rest_framework.validators import UniqueForDateValidator as UniqueForDateValidator
+from rest_framework.validators import UniqueForMonthValidator as UniqueForMonthValidator
+from rest_framework.validators import UniqueForYearValidator as UniqueForYearValidator
+from rest_framework.validators import UniqueTogetherValidator as UniqueTogetherValidator
```

Comment:
Why do we need to export validators from serializers? Is it documented somewhere?

Reply:
I thought `serializers.UniqueTogetherValidator` is a valid syntax, similar to `serializers.CharField`. But it only applies to `fields` and `relations`, imported under [this line](https://github.com/encode/django-rest-framework/blob/a677b09/rest_framework/serializers.py#L47) :facepalm: